vscode/src/commands.ts

```diff
@@ -11,48 +13,59 @@ function getFullScreenTab() {
   return tabs.find((tab) => (tab.input as any)?.viewType?.endsWith("konveyor.konveyorGUIView"));
 }
 
-const commandsMap: (
-  extensionContext: vscode.ExtensionContext,
-  state: ExtensionState,
-) => {
+const commandsMap: (state: ExtensionState) => {
```

Comment:
if `any` is required here (i.e. due to vs code interface) then let's silence the linters  - GH actions create a lot of noise here (the msg is multiplied by platforms):
![image](https://github.com/user-attachments/assets/6a8ca645-5c57-45bc-a137-613ea0c924f7)